activerecord/lib/active_record/core.rb

```diff
@@ -279,6 +279,14 @@ def arel_table # :nodoc:
         @arel_table ||= Arel::Table.new(table_name, type_caster: type_caster)
       end
 
+      # Returns an instance of <tt>Arel::Attributes::Attribute</tt> which represents the named column.
+      # For example:
+      #
+      #   Developer.where(Developer[:salary].not_eq(9000))
+      def [](name)
+        arel_table[name]
+      end
+
       def arel_attribute(name, table = arel_table) # :nodoc:
```

Comment:
I suppose `[]` should be the alias for `arel_attribute`.

Reply:
Good point, I should have read the code 😅

Reply:
🤔 so if you had a model `Weird` and you did `Weird[:id]` that would return the column?